src/status.router.ts

```diff
@@ -35,3 +36,27 @@ statusRouter.get('/', async (_req: Request, res: Response) => {
 
   res.status(200).send(status)
 })
+
+statusRouter.get('/vars', async (_req: Request, res: Response) => {
+  const mints = () => {
+    const {
+      MOBILE_MINT: mobile,
+      IOT_MINT: iot,
+      HNT_MINT: hnt,
+      DC_MINT: dc,
+    } = process.env
+    return { mobile, iot, hnt, dc }
+  }
+
+  const metadata_urls = () => {
```

Comment:
do these need to be functions or could these be declared as consts?

Reply:
either way. I guess a const is probably more readable, will update.